Сluster build error - x509: certificate relies on legacy Common Name field

Error building cluster - `x509: certificate relies on legacy Common Name field, use SANs or temporarily enable Common Name matching with GODEBUG=x509ignoreCN=0`

### system log:
```
Jul 29 12:13:24 ip-172-31-31-133 rkt[1486]: [ 2437.262873] vault[6]: 2021-07-29T12:13:24.453Z [INFO]  http: TLS handshake error from 172.31.31.151:56356: remote error: tls: bad certificate
Jul 29 12:13:24 ip-172-31-31-133 rkt[1486]: [ 2436.986806] vault[6]: 2021-07-29T12:13:24.177Z [ERROR] core: failed to retry join raft cluster: retry=2s
Jul 29 12:13:24 ip-172-31-31-133 rkt[1486]: [ 2436.986384] vault[6]: 2021-07-29T12:13:24.177Z [WARN]  core: join attempt failed: error="error during raft bootstrap init call: Put "https://node2.vault.int:8200/v1/sys/storage/raft/bootstrap/challenge": x509: certificate relies on legacy Common Name field, use SANs or temporarily enable Common Name matching with GODEBUG=x509ignoreCN=0"
Jul 29 12:13:24 ip-172-31-31-133 rkt[1486]: [ 2436.981194] vault[6]: 2021-07-29T12:13:24.171Z [INFO]  core: attempting to join possible raft leader node: leader_addr=https://node2.vault.int:8200
Jul 29 12:13:24 ip-172-31-31-133 rkt[1486]: [ 2436.980500] vault[6]: 2021-07-29T12:13:24.170Z [INFO]  core: security barrier not initialized
Jul 29 12:13:24 ip-172-31-31-133 rkt[1486]: [ 2436.979924] vault[6]: 2021-07-29T12:13:24.170Z [WARN]  core: join attempt failed: error="error during raft bootstrap init call: Put "https://node1.vault.int:8200/v1/sys/storage/raft/bootstrap/challenge": x509: certificate relies on legacy Common Name field, use SANs or temporarily enable Common Name matching with GODEBUG=x509ignoreCN=0"
Jul 29 12:13:24 ip-172-31-31-133 rkt[1486]: [ 2436.974437] vault[6]: 2021-07-29T12:13:24.164Z [INFO]  core: attempting to join possible raft leader node: leader_addr=https://node1.vault.int:8200
```
the error occurs on the latest **Vault** version `1.7.3`, with version `1.4.2` works fine

### how to reproduce:

```terraform
module "vault" {
  source = "github.com/binlab/terraform-aws-vault-ha-raft?ref=v0.1.8"

  cluster_name        = "vault-raft"
  cluster_count       = 3
  node_instance_type  = "t3a.small"
  autounseal          = true
  nat_enabled         = true
  vpc_cidr            = "172.31.31.0/24"

  vpc_public_subnets = [
    "172.31.31.0/28",
    "172.31.31.16/28",
    "172.31.31.32/28",
  ]

  vpc_private_subnets = [
    "172.31.31.128/28",
    "172.31.31.144/28",
    "172.31.31.160/28",
  ]

  ami_image  = "ami-0bb5fc1412bbbb988"
  docker_tag = "1.7.3"
}
```
the same behavior with latest **Vault** module code from master where `rkt` replaced with a `Docker`

Logs with a latest code from `master` with a `Docker` instead of `rkt`
```shell
Jul 29 12:23:22 ip-172-31-31-136.ec2.internal docker[1760]: 2021-07-29T12:23:22.251Z [INFO]  http: TLS handshake error from 172.31.31.168:55606: remote error: tls: bad certificate
Jul 29 12:23:22 ip-172-31-31-136.ec2.internal docker[1760]: 2021-07-29T12:23:22.090Z [INFO]  core: security barrier not initialized
Jul 29 12:23:22 ip-172-31-31-136.ec2.internal docker[1760]: 2021-07-29T12:23:22.044Z [ERROR] core: failed to retry join raft cluster: retry=2s
Jul 29 12:23:22 ip-172-31-31-136.ec2.internal docker[1760]: 2021-07-29T12:23:22.044Z [WARN]  core: join attempt failed: error="error during raft bootstrap init call: Put "https://node2.vault.int:8200/v1/sys/storage/raft/bootstrap/challenge": x509: certificate relies on legacy Common Name field, use SANs or temporarily enable Common Name matching with GODEBUG=x509ignoreCN=0"
Jul 29 12:23:22 ip-172-31-31-136.ec2.internal docker[1760]: 2021-07-29T12:23:22.038Z [INFO]  core: attempting to join possible raft leader node: leader_addr=https://node2.vault.int:8200
Jul 29 12:23:22 ip-172-31-31-136.ec2.internal docker[1760]: 2021-07-29T12:23:22.038Z [INFO]  core: security barrier not initialized
Jul 29 12:23:22 ip-172-31-31-136.ec2.internal docker[1760]: 2021-07-29T12:23:22.038Z [WARN]  core: join attempt failed: error="error during raft bootstrap init call: Put "https://node1.vault.int:8200/v1/sys/storage/raft/bootstrap/challenge": x509: certificate relies on legacy Common Name field, use SANs or temporarily enable Common Name matching with GODEBUG=x509ignoreCN=0"
Jul 29 12:23:22 ip-172-31-31-136.ec2.internal docker[1760]: 2021-07-29T12:23:22.031Z [INFO]  core: attempting to join possible raft leader node: leader_addr=https://node1.vault.int:8200
```

--- a/tls.tf
+++ b/tls.tf
@@ -71,10 +71,14 @@ resource "tls_cert_request" "node" {
   private_key_pem = tls_private_key.node[count.index].private_key_pem
 
   subject {
-    common_name         = format(local.internal_domain_tmpl, count.index)
+    common_name         = format("%s Node %d", var.cluster_description, var.cluster_count)
     organizational_unit = format("%s Certificate", var.cluster_description)
     organization        = var.cluster_description
   }
+
+  dns_names = [
+    format(local.internal_domain_tmpl, count.index)
+  ]
 
   lifecycle {
     prevent_destroy = false

